sklearn/preprocessing/discretization.py

```diff
@@ -32,6 +33,18 @@ class KBinsDiscretizer(BaseEstimator, TransformerMixin):
         Column indices of ignored features. (Example: Categorical features.)
         If ``None``, all features will be discretized.
 
+    encode : string {'onehot', 'onehot-dense', 'ordinal'} (default='ordinal')
```

Comment:
I don't thing you need the type information when you list all possible values. Double-check with the numpy doc style.

Reply:
@lesteve It seems that this is not the case in many functions (e.g. pca, LinearSVC) and I have no idea how to check the doc style. Could you please help me? Thanks.

Reply:
From https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections

```
When a parameter can only assume one of a fixed set of values, those values can be listed in braces, with the default appearing first:

order : {'C', 'F', 'A'}
    Description of `order`.
```